internal/testutil/testutil.go

```diff
@@ -79,7 +86,8 @@ func NewManager(t *testing.T) *Manager {
 			filepath.Join(root, "api", "v1", "config", "crd"),
 			testCrdDir,
 		},
-		ErrorIfCRDPathMissing: true,
+		ErrorIfCRDPathMissing:    true,
+		AttachControlPlaneOutput: os.Getenv("ACTIONS_RUNNER_DEBUG ") != "" || os.Getenv("ACTIONS_STEP_DEBUG ") != "",
```

Comment:
Are the trailing spaces needed in the env var names?

Reply:
Nope! Fixed.